release-notes/README.md

```diff
@@ -4,11 +4,17 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## May 18, 2022
+
+### Android Campaign Classic 1.0.2
+
+* Android Campaign Classic SDK is now Adobe Campaign Classic (ACC) v8 compatible! Broadlog ID can be provided in the UUID format in the notification tracking APIs.
+
 ## May 17, 2022
 
 ### iOS Campaign Classic 2.1.1
 
-* ACPCampaignClassic iOS SDK is now Adobe Campaign Classic (ACC) v8 compatible! BroadlogId can be provided in the UUID format in the notification tracking APIs.
+* ACPCampaignClassic iOS SDK is now Adobe Campaign Classic (ACC) v8 compatible! Broadlog ID can be provided in the UUID format in the notification tracking APIs.
```

Comment:
missed this, this should be Android and not iOS

Reply:
That's for iOS, so it's iOS. I updated BroadlogId -> Broadlog ID for both iOS/ Android release notes

Reply:
ah, whoops. looks good to me.